collector.go

```diff
@@ -110,26 +112,40 @@ func NewCwCollector(target string, taskName string, region string) (*cwCollector
 			Name: "cloudwatch_exporter_scrape_duration_seconds",
 			Help: "Time this CloudWatch scrape took, in seconds.",
 		}),
+		ScrapeTimeBuckets: prometheus.NewHistogram(prometheus.HistogramOpts{
+			Name: "cloudwatch_exporter_scrape_duration_seconds_buckets",
+			Help: "Time this CloudWatch scrape took, in seconds and shown in buckets",
```

Comment:
You're missing the buckets. I suggest those values:
```suggestion
			Help: "Time this CloudWatch scrape took, in seconds and shown in buckets",
			Buckets: []float64{0.1, 0.25, 0.5, 1., 5., 8., 10., }
```

Reply:
Done :)